Objects/boolobject.c

```diff
@@ -13,13 +13,19 @@ bool_repr(PyObject *self)
 {
     PyObject *s;
 
-    if (self == Py_True)
-        s = true_str ? true_str :
-            (true_str = PyUnicode_InternFromString("True"));
-    else
-        s = false_str ? false_str :
-            (false_str = PyUnicode_InternFromString("False"));
-    Py_XINCREF(s);
+    if (self == Py_True) {
+        if (_PY_ONCEVAR_INIT(true_str, PyUnicode_InternFromString("True"))) {
```

Comment:
If make `_PY_ONCEVAR_INIT()` returning the value (NULL in case of error) the code would be simpler.

```c
if (self == Py_True) {
    s = _PY_ONCEVAR_INIT(true_str, PyUnicode_InternFromString("True"));
}
else {
    s = _PY_ONCEVAR_INIT(false_str, PyUnicode_InternFromString("False"));
}
Py_XINCREF(s);
return s;
```